src/status_im/contexts/communities/actions/addresses_for_permissions/events.cljs

```diff
@@ -22,7 +22,9 @@
     [:cofx :schema.re-frame/cofx]
     [:args [:schema [:? [:catn [:community-id :string]]]]]]
    [:map {:closed true}
-    [:fx [:vector {:min 1 :max 1} :schema.common/rpc-call]]]])
+    [:fx
+     [:tuple
+      [:tuple [:= :json-rpc/call] :schema.common/rpc-call]]]]])
```

Comment:
Should we be rigid about the returned effects map schema? From instrumenting events, I see most value from the arguments schema and by instrumenting the args from `:json-rpc/call` (or any other event/effect), we wouldn't need to check the args everywhere it's used. WDYT?

Reply:
Good point. There's a balance to strike. We have some events checking only that `:fx` is a map. Here I wanted to make sure it's nothing more than a tuple of tuples to experiment a little bit. I don't think this should be done all the time.

But I think the more we spec events, the more we should think about coming up with a schema representing re-frame's effects map. Our guidelines recommend using only `:db` and `:fx` in the returned map, so we could reinforce this idea more generally.

This generality won't be able to check something like I did OTOH, it will be more "open", but I think that's okay.

Reply:
Agreed, enforcing the guideline through the schema here would be more useful than the specific events and effects that are returned 👍. Can we instrument the `:json-rpc/call` effect instead?

Reply:
Oh definitely, we should instrument all highly used utilities/functions IMO. Those are the ones where it pays off to have schemas.

We had an agreement in the beginning to not spec non-component stuff, but time has passed and the team seems to be okay with Malli and a few are even enjoying. I think it's a good time to start already.

Reply:
> Can we instrument the :json-rpc/call effect instead?

We can instrument the function that gets called, but in re-frame, events just return data, so we need to instrument the event handlers themselves because the `status-im.common.json-rpc.events/call` is never called directly.